force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,8 +29,13 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+         if (x > y) {
+             return 'Hello World!';
+         } 
+         return null;
```

Comment:
Consider checking for invalid values first before you evaluate the desired expression. Return early.